src/components/MyData/LineChart/TraceLine.tsx

```diff
@@ -32,7 +32,7 @@ export const TraceLine = (props: Props) => {
   const yDomain = [domainMin, domainMax] as [number, number];
 
   const trend = useMemo(() => {
-    if (showTrend) {
+    if (showTrend && data.length > 1) {
```

Comment:
We should only show the trend if it has 2+ points. A single point produces a flat trend which isn't meaningful and 0 points causes a NaN error because of division by zero.